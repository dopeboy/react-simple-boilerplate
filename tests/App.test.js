import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import App from '../src/App';
import './setup';

test('should render', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("countains text 'Hello World!'", () => {
  const app = shallow(<App />);
  expect(app.text()).toEqual('Hello World!');
});
