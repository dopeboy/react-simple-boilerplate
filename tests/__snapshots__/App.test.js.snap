// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App 1`] = `
<div>
  Hello World!
</div>
`;

exports[`should render 1`] = `
<div>
  Hello World!
</div>
`;
